openshift/jenkins/Jenkinsfile.promote-to-test

```diff
@@ -0,0 +1,143 @@
+#!groovy
+
+// ------------------
+// Pipeline Variables
+// ------------------
+String[] DEPLOYMENT_ENVIRONMENT_TAGS = ['dev', 'test', 'prod']
```

Comment:
References to PROD useful for later addition of prod to this pipeline? Or would that be in another Jenkinsfile.

Reply:
I'll remove the prod reference